app/views/layouts/application.html.slim

```diff
@@ -15,6 +15,10 @@
     javascript:
       gaTracker.init('#{ENV.fetch("GOOGLE_ANALYTICS_ID")}');
 
+  - if controller.controller_name == 'claims'
```

Comment:
one of the problems with this approach as it stands is that the review page would also require a session prompt and that is rendered through a different controller